packages/com.circleci.v2/PklProject

```diff
@@ -17,5 +17,5 @@
 amends "../basePklProject.pkl"
 
 package {
-  version = "1.1.1"
+  version = "1.1.2"
```

Comment:
nit; in the strictest sense, isn't this `1.2.0`?

Reply:
It's a bugfix though?